include/parser/stream.h

```diff
@@ -125,6 +234,19 @@ class StreamParser {
     return res;
   }
 
+  std::vector<uint8_t> getBytesUntil(const QString &name, uint8_t termination,
+                                     bool include_termination = true) {
+    auto data =
+        getDataUntil(name, data::RepackFormat{data::RepackEndian::LITTLE, 8},
+                     data::BinData::fromRawData(8, {termination}),
+                     data::FieldHighType(), include_termination);
+    std::vector<uint8_t> res;
+    for (size_t i = 0; i < data.size(); i++) {
+      res.push_back(data.element64(i));
```

Comment:
Is this really how we're supposed to use BinData? Feels like we're doing a lot of unnecessary stuff just to pack BinData.rawData into uint64_t, so we can take it out again. 

Also I would expect to get compiler warnings in here (implicit cast from uint64_t to uint8_t); at the very least can we add an explicit cast?